docs/source/version_compatibility.rst

```diff
@@ -0,0 +1,34 @@
+**************
+Sabre Versions
+**************
+
+The following table explains what versions of the Sawtooth Sabre SDK are
```

Comment:
Recommended rewording:

The following table shows the compatible versions of the Sabre transaction processor, Sabre SDK, and Sawtooth Rust SDK. It also shows the Docker tag for the Sabre transaction processor image.